contracts/colony/ColonyFunding.sol

```diff
@@ -95,24 +95,34 @@ contract ColonyFunding is ColonyStorage, PatriciaTreeProofs { // ignore-swc-123
     }
   }
 
+  int256 constant MAX_PAYOUT_MODIFIER = int256(WAD);
+  int256 constant MIN_PAYOUT_MODIFIER = -int256(WAD);
+
   function claimExpenditurePayout(uint256 _id, uint256 _slot, address _token) public
   stoppable
   expenditureExists(_id)
   expenditureFinalized(_id)
   {
     Expenditure storage expenditure = expenditures[_id];
     ExpenditureSlot storage slot = expenditureSlots[_id][_slot];
-    require(add(expenditure.finalizedTimestamp, slot.claimDelay) <= now, "colony-expenditure-cannot-claim");
+
+    require(
+      add(expenditure.finalizedTimestamp, add(expenditure.globalClaimDelay, slot.claimDelay)) <= now,
+      "colony-expenditure-cannot-claim"
```

Comment:
Oh good point! I hadn't thought about that interaction. I suppose setting them to MAX / 3 would be the answer here (since timestamp + delay + delay).

And yes, a mega multi-test would probably be appropriate.